frontend2/src/components/Header.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface headerProps {
+    loggedIn: boolean
+    logoPath: string
+}
+const Header = ({loggedIn, logoPath}: headerProps): JSX.Element => {
```

Comment:
for this too, let's be consistent with how we defined basecard
`const BaseCard: React.FC<BaseCardProps> = ({ GameImage, GameName, GameDescr, GameYear }) => {`